### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by sumnerevans**
Updated README:

- consistent 80 character line limit
- grammar updates
- link to AUR package for Arch Linux users

### CODE REVIEW

**Comment by rhysd** [comment]
Thank you for updates! They're helpful because I'm not a native English speaker. In Japanese, an article like 'a' or 'the' doesn't exist. So it's hard to use them properly for me...

File: README.md [hunk 1]
```diff
@@ -230,21 +258,21 @@ By default, some key shortcuts for tab items are set in addition to above table.
...
+If you add multiple accounts to the `accounts` array in `config.json`, an
+`Accounts` menu item will appear in the application menu.
 
 ![multi account menu item](https://github.com/rhysd/ss/blob/master/Mstdn/multi-account.png?raw=true)
 
-It will show the list of your account. Check mark is added for current user.
-When you click menu item of non-current user, application window will be recreated and switch page
-to the account.
+It will show the list of your accounts. The check mark indicates the current
+user.  When you click menu item of non-current user, application window will be
```
Comment (rhysd) [inline README.md]: Number of spaces after period should be one ('user. When...')

**Review by rhysd — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sumnerevans) [reply README.md]: Good catch. Fixed!

**Review by rhysd — APPROVED** [review]

**Comment by rhysd** [comment]
Merged. Thank you.

**Pull request merged**